### PR TITLE
updated demo to walkthrough changing retention policy on archive action

### PR DESCRIPTION
**Pull request by cpvanfossan**

### CODE REVIEW

File: docs/demo_guide.md [hunk 1]
```diff
@@ -241,25 +241,31 @@ DEMO B: Running an Archive Rule
     the rule is enabled. This will enable you to have an **automated retention
     policy** in Relativity.
     
-    > **NOTE:** For this demo retention has be purposely set to 0 hours, which means that all of the `non-alerted` data will be
```
Comment (mishakogan) [inline docs/demo_guide.md:244]: Retention is no longer set to zero? Curious why we removed the note? @cpvanfossan

Comment (mishakogan) [reply docs/demo_guide.md:244]: Never mind, I see another bullet with this info

Comment (cpvanfossan) [reply docs/demo_guide.md:244]: This is the issue I was having with hopper. Every time I deploy the image, the retention policy is being set to 168 hours.

**Comment by mishakogan** [comment]
This is a bit suspicious. What resets the number? If template has it set to
zero, it should stay as zero.

Something to double check

File: docs/demo_guide.md (hunk 1 above)
Comment (mishakogan) [reply docs/demo_guide.md:244]: This is a bit suspicious. What resets the number? If template has it set to zero, it should stay as zero.

Something to double check

**Comment by cpvanfossan** [comment]
Resolved hopper issue - we can keep the demo as it was.

**Pull request closed without merging**